### ISSUE TITLE
Create repository of Sails plugins

### ISSUE THREAD

**Issue by dbkaplun** [balderdashy/sails#2868]
It's awesome that SailsJS has plugin support now. However I can't find any plugins for Sails! Have you thought of keeping a list somewhere?

**Comment by tjwebb** [comment]
Yes, they are searchable through npm, but this is something we don't currently have. It's planned, but we just haven't had the time as of yet. If there's a particular thing you're looking for, for now, you can try asking in the gitter channel: https://gitter.im/balderdashy/sails

**Comment by CWyrtzen** [comment]
Anyone want to work on documentation for this? ;)

**Comment by sailsbot** [comment]
Thanks for posting, @dbkaplun.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
